packages/ubuntu_bootstrap/lib/widgets/option_button.dart

```diff
@@ -1,8 +1,7 @@
 import 'package:flutter/material.dart';
-import 'package:ubuntu_bootstrap/pages/try_or_install/try_or_install_model.dart';
 import 'package:yaru_widgets/yaru_widgets.dart';
 
-class OptionButton extends StatelessWidget {
```

Comment:
Maybe it actually is trivial to convert those tile widgets that I created to use this one...